vue.config.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
     proxy: {
       // this is needed to prevent a CORS error when running locally
       '/local-keycloak-config-url/*': {
-        target: 'https://business-edit-dev.apps.silver.devops.gov.bc.ca/businesses/edit/config/kc/',
+        target: 'https://business-create-dev.apps.silver.devops.gov.bc.ca/businesses/edit/config/kc/',
```

Comment:
Yikes, good fix :)

Reply:
haha yes, for now! It might change tomorrow though 😆

Reply:
Actually you're right... 🙄... Patrick and I were talking about using external (vanity) URLs for Create and Edit UIs.

Nevertheless, thanks for fixing my copy-paste bug 😛